api/mymonero_rest.md

```diff
@@ -0,0 +1,391 @@
+| mempool        | boolean          | True if tx is in mempool  |
+| mixin          | uint32           | Mixin of the receive      |
+
+> `id` is determined by the monero daemon. It is the offset that a
+> transaction appears in the blockchain from the genesis block.
+
+> `timestamp` and `height` are not sent when `mempool` is true.
+
+> `hash` is determined by how the monero core computes the hash.
+
+> `spent_outputs` is the list of possible spends in _this_ transaction only.
+
+> `payment_id` is omitted if the transaction had none. It is decrypted when the
+> encrypted form is used. The decryption may be incorrect - if the transaction
+> was TO another address, then this will be random bytes. This happens
+> frequently with outgoing payment ids; the received XMR in the transaction is
+> change and the payment id is for the real recipient.
+
+> `mixin` does not include the real spend - this is the number of dummy inputs.
+
+**uint16** / **uint32** / **uint64**
+
+Sent as a standard decimal encoded number in JSON. The JSON decoder must reject
+number values that exceed the specified bit-width.
+
+**uint64-string**
+
+A uint64 encoded as a decimal string value in JSON. Used when a value may
+exceed 2^53 - all numbers are 64-bit floats in JavaScript.
+
+**random_output** object
+
+|     Field    |      Type     |            Description             |
+|--------------|---------------|------------------------------------|
+| global_index | uint64-string | Index within amount                |
+| public_key   | bytes         | Bytes of output public key         |
+| rct          | bytes         | Bytes containing ringct commitment |
+
+> `global_index` is determined by the monero daemon. It is the offset from the
+> first time the amount appeared in the blockchain. After ringct, this is the
+> order of outputs as they appear in the blockchain.
+
+**random_outputs** object
+
+Randomly selected outputs for use in a ring signature.
+
+|   Field   |            Type          |       Description       |
+|-----------|--------------------------|-------------------------|
+|  amount   |       uint64-string      | XMR amount, 0 if ringct |
+| outputs * | array of random_output's | Selected outputs        |
+
+> `outputs` is omitted by the server if the `amount` does not have enough
+> mixable outputs.
+
+### Methods
+#### `get_address_info`
+Returns the minimal set of information needed to calculate a wallet balance.
+The server cannot calculate when a spend occurs without the spend key, so a
+list of candidate spends is returned.
+
+**Request** object
+
+|   Field   |      Type      |            Description                |
+|-----------|----------------|---------------------------------------|
+| address   | base58-address | Address to retrieve                   |
+| view_key  | binary         | View key bytes for authorization      |
+
+> If `address` is not authorized, the server must return a HTTP 405
```

Comment:
Why should not it be allowed? The HTTP code for Forbidden error is 403. (405 - Method not allowed). Should we keep 405 or 403?